CRM/Financial/DAO/PaymentProcessorType.php

```diff
@@ -209,6 +210,7 @@ public static function &fields() {
           'type' => CRM_Utils_Type::T_STRING,
           'title' => ts('Payment Processor Title'),
           'description' => ts('Payment Processor Name.'),
+          'required' => TRUE,
```

Comment:
There's an obvious typo not caused by this PR but can we fix please - description "Payment Processor Title".  In fact, shouldn't these be "Payment Processor Type Name" and "Payment Processor Type Title"?